src/uu/mv/src/mv.rs

```diff
@@ -103,6 +104,14 @@ static OPT_VERBOSE: &str = "verbose";
 static OPT_PROGRESS: &str = "progress";
 static ARG_FILES: &str = "files";
 
+/// Returns true if the passed `path` ends with a path terminator.
+fn path_ends_with_terminator(path: &Path) -> bool {
+    path.as_os_str()
+        .as_bytes()
```

Comment:
I guess you have seen the error that there is no `as_bytes` function under Windows?

Reply:
Lord no I haven't, thanks for pointing it out.

Reply:
Any tips on how I can develop against windows on linux? I'm using vscode with rust-analyzer. Tried fiddling with the target and features but couldn't get what I wanted: for the IDE to think that I was on windows.

Reply:
@cakebaker can I run the workflows locally? They won't run until they're approved: 

![Uploading image.png…]()

Reply:
@mickvangelderen the approval is no longer needed and the workflows should run automatically in the future.